agr_java_core/src/main/java/org/alliancegenome/api/entity/PhenotypeAnnotationDocument.java

```diff
@@ -0,0 +1,81 @@
+package org.alliancegenome.api.entity;
+
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.alliancegenome.curation_api.model.entities.*;
+import org.alliancegenome.curation_api.model.entities.ontology.ECOTerm;
+import org.alliancegenome.es.index.site.document.SearchableItemDocument;
+import org.apache.commons.collections4.CollectionUtils;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class PhenotypeAnnotationDocument extends SearchableItemDocument {
```

Comment:
Does this need to extend SearchableItemDocument it would be better to just add category here and extend ESDocument.